docker/Dockerfile.cpu

```diff
@@ -1,12 +1,26 @@
-FROM awsdeeplearningteam/mms_cpu_base:latest
+FROM ubuntu:latest
 
-RUN pip install mxnet-model-server \
-    && pip uninstall --yes mxnet \
-    && pip install mxnet-mkl \
-    && mkdir /mxnet_model_server
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    build-essential \
+    libatlas-base-dev \
+    libopencv-dev \
+    graphviz \
```

Comment:
Why we need all these but without mxnet? In future mms will not install mxnet.

Reply:
Oh, regarding why MXNet is not a part of this docker, its because its a part of "mxnet-model-server"s pip install